wp-content/themes/illdy-child/layout/js/translations_page.js

```diff
@@ -157,6 +172,19 @@ var TranslationsPage = (function(window, $) {
       }
     };
 
+    this.setSelectedResource = function(selected) {
+        var resourceFilter = $("#lang-list-resource-filter")[0];
+        var options = resourceFilter.options;
+        for (var i = 0; i < options.length; i++) {
+            if (options[i].value === selected) {
+                resourceFilter.value = selected;
+                resourceFilter.onclick();

```

Comment:
Perhaps the handler should be running onChange instead of onClick because we are worried about when the value changes right? Not just when someone clicks it.

Reply:
Unfortunately onChange() doesn't work consistently in IE. `onClick()` does though.  :(